ImperatorToCK3/Data_Files/blankMod/dummy.txt

```diff
@@ -1 +0,0 @@
-I'm a dummy file. My purpose is to ensure my directory is created.
```

Comment:
Was this file left? You should be able to fully delete it now that there are other files.

Reply:
It's removed